code/modules/uplink/uplink_item.dm

```diff
@@ -222,6 +222,16 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	item = /obj/item/weapon/gun/ballistic/automatic/c20r/unrestricted
 	include_modes = list(/datum/game_mode/gang)
 
+/datum/uplink_item/dangerous/minigun
+	name = "M134 Minigun"
+	desc = "She weighs one hundred fifty kilograms and fires two hundred dollar, custom-tooled cartridges at ten rounds per second."
+	item = /obj/item/weapon/minigunpack/ballistic
+	surplus = 0
+	cant_discount = TRUE	//No.
+	player_minimum = 40		//No lowpop minigunning.
+	include_modes = list(/datum/game_mode/nuclear)
+	cost = 55
```

Comment:
I don't see this ever being at a cost that makes sense. You're better off buying a mech or a cyborg if you're going to be spending this much money on a single item since this offers no survivability, and the SAW fits in the same category of weapon but at 20tc cheaper